### DISCUSSION TITLE
App crash in reducer

### DISCUSSION THREAD

**Discussion by markdevios** [pointfreeco/swift-composable-architecture#3350] [Q&A]
I am using The Composable Architecture in my application. I am appending large number of paths in navigation stack, which could not be avoided. So i am facing stack overflow issue. To overcome the issue, i have used COW and splitting up the reducer. But still I am facing a crash after few screens. Couldn’t able to address the exact crash location, as it points only to reducer, if app is crashed.

**Comment by stephencelis** [comment]
@markdevios Can you share an example stack trace? Are you using the `@Presents` and `PresentationAction` presentation tools?
